lib/graphql/introspection/input_value_type.rb

```diff
@@ -7,11 +7,7 @@
   field :defaultValue, types.String, "The value applied if no other value is provided" do
     resolve -> (obj, args, ctx) {
       value = obj.default_value
-      if value.is_a?(String)
-        "\"#{value}\""
-      else
-        value
-      end
+      !value.nil? ? JSON.dump(obj.type.coerce_result(value)) : nil
```

Comment:
This is a lot of calls on one line ... could we at least remove one by reversing the boolean logic? 

```ruby
value.nil? ? nil : JSON.dump(...)
```

?